movie-showcase/src/routes/display/+page.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+    import Carousel from 'svelte-carousel'	
+    import { writableArray } from '../../stores';
+    import { slideDuration } from '../../stores';
+    import { carouselItemCount } from '../../stores';    
+    import { onMount } from "svelte";
+
+    onMount(checkSettings);
+
+    function checkSettings(){
+        if($slideDuration == 0 || $slideDuration == null) $slideDuration = 5;
+        if($carouselItemCount == 0 || $carouselItemCount == null) $carouselItemCount = 1;
+    }
+</script>
+
+{#if $writableArray.length >= 1} 
+    <Carousel
+    particlesToShow={$carouselItemCount}
+    autoplay
+    autoplayDuration={0}
+    duration={$slideDuration * 1000}
+    timingFunction="linear"
+    dots={false}
+    arrows={false}
+    swiping={false}
+    >
+
+    {#each $writableArray as item}
+    <div id="card">
```

Comment:
Ids have to be unique on a page, since this is in a loop it's not very likely to be unique and a class would be more appropriate